spec/rewrite/client_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require_relative '../spec_helper'
+
+describe EasyPost::Client do
+  describe 'client object' do
+    it 'create a client object timeouts' do
+      client = described_class.new(api_key: 'fake_api_key', read_timeout: 10, open_timeout: 1)
+
+      expect(client.instance_variable_get('@configuration').instance_variable_get('@read_timeout')).to eq(10)
+      expect(client.instance_variable_get('@configuration').instance_variable_get('@open_timeout')).to eq(1)
```

Comment:
issue: like other libs, we want to be able to pull out properties of a client directly, for instance, I should be able to just print out `client.read_timeout` directly. By consolidating the client and configuration classes, this should be resolved

Reply:
fixed